Plugins/Flow.Launcher.Plugin.Program/Logger/ProgramLogger.cs

```diff
@@ -20,36 +20,13 @@ internal static class ProgramLogger
     {
         public const string DirectoryName = "Logs";
 
-        static ProgramLogger()
-        {
-            var path = Path.Combine(DataLocation.DataDirectory(), DirectoryName, Constant.Version);
-            if (!Directory.Exists(path))
-            {
-                Directory.CreateDirectory(path);
-            }
-
-            var configuration = new LoggingConfiguration();
-            var target = new FileTarget();
-            configuration.AddTarget("file", target);
-            target.FileName = path.Replace(@"\", "/") + "/${shortdate}.txt";
-#if DEBUG
-            var rule = new LoggingRule("*", LogLevel.Debug, target);
-#else
-            var rule = new LoggingRule("*", LogLevel.Error, target);
-#endif
-            configuration.LoggingRules.Add(rule);
-            LogManager.Configuration = configuration;
-        }
-
```

Comment:
It is removed because it will override the config setting in Log.cs (which is the same in before so no effect is shown).